tests/functional/test_loaded_model.py

```diff
@@ -0,0 +1,63 @@
+import numpy as np
+import grpc
+import pytest
+
+from tritonclient.grpc import service_pb2
+from tritonclient.grpc import service_pb2_grpc
+import tritonclient.grpc as grpcclient
+from model_config import MODEL_CONFIG
+PORT=8111
+
+
+@pytest.mark.parametrize("model", MODEL_CONFIG.keys())
+def test_inference(model):
+    triton_client = grpcclient.InferenceServerClient(
+        url=f"localhost:{PORT}",
+        verbose=False)
+
+    config = MODEL_CONFIG[model]
+
+    inputs = []
+    inputs.append(grpcclient.InferInput(config["input_name"], config["shape"], "FP32"))
+
+    inputs[0].set_data_from_numpy(np.ones(config["shape"], dtype=np.float32))
+    results = triton_client.infer(
+        model_name= model,
+        inputs=inputs)
+    output = results.as_numpy(config["output_name"])
+    assert output.shape == config["expected_shape"]
+
+def test_dynamic_shape_inference():
+    triton_client = grpcclient.InferenceServerClient(
+        url=f"localhost:{PORT}",
+        verbose=False)
```

Comment:
this might be useful for debugging, maybe add the ability to set True via an environment variable?